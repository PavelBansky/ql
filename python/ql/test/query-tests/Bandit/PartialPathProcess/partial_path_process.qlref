Bandit/partial_path_process.ql