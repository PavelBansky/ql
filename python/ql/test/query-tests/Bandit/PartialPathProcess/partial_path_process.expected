| Test.py:3:1:3:33 | pop() | Starting a process with a partial executable path |
| Test.py:5:1:5:21 | pop() | Starting a process with a partial executable path |
| Test.py:7:1:7:30 | pop() | Starting a process with a partial executable path |
| Test.py:8:1:8:35 | pop() | Starting a process with a partial executable path |
